src/js/controller/model.js

```diff
@@ -384,8 +390,8 @@ define([
 
         };
 
-        this.persistVideoSubtitleTrack = function(trackIndex) {
-            this.setVideoSubtitleTrack(trackIndex);
+        this.persistVideoSubtitleTrack = function(trackIndex, tracks) {
+            this.setVideoSubtitleTrack(trackIndex, tracks);
             this.persistCaptionsTrack();
```

Comment:
Why does persistCaptionsTrack set the menu to "Off"? That should be the default on playlist item. It should not be set back to off based on tracks. it would be off already.